packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/multi-party.js

```diff
@@ -0,0 +1,166 @@
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@ciscospark/plugin-phone';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import handleErrorEvent from '../lib/handle-error-event';
+import {browserOnly, maxWaitForEvent} from '@ciscospark/test-helper-mocha';
+
+if (process.env.NODE_ENV !== `test`) {
+  throw new Error(`Cannot run the plugin-phone test suite without NODE_ENV === "test"`);
+}
+
+browserOnly(describe)(`plugin-phone`, function() {
+  this.timeout(60000);
+
+  describe(`Phone`, () => {
+    let mccoy, spock;
+    before(`create users and register`, () => testUsers.create({count: 2})
+      .then((users) => {
+        [mccoy, spock] = users;
+        spock.spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          }
+        });
+
+        mccoy.spark = new CiscoSpark({
+          credentials: {
+            authorization: mccoy.token
+          }
+        });
+
+        return Promise.all([
+          spock.spark.phone.register(),
+          mccoy.spark.phone.register()
+        ]);
+      }));
+
+    let ringMccoy;
+
+    beforeEach(() => {
+      ringMccoy = sinon.spy();
+      mccoy.spark.phone.on(`call:incoming`, ringMccoy);
+    });
+
+    beforeEach(() => {
+      spock.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+      mccoy.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+    });
+
+    afterEach(() => {
+      spock.spark.config.phone.enableExperimentalGroupCallingSupport = false;
+      mccoy.spark.config.phone.enableExperimentalGroupCallingSupport = false;
+    });
+
+    after(`unregister spock and mccoy`, () => Promise.all([
+      spock && spock.spark.phone.deregister()
+        .catch((reason) => console.warn(`could not disconnect spock from mercury`, reason)),
+      mccoy && mccoy.spark.phone.deregister()
+        .catch((reason) => console.warn(`could not disconnect mccoy from mercury`, reason))
+    ]));
+
+
+    describe(`#dial()`, () => {
+      it(`calls a room by hydra room id`, () => spock.spark.request({
```

Comment:
Do we want to test a room with more than 2 members? I know it isn't technically a 1:1, but it seems like it might behave that way with calling.

Reply:
There's another test in the next PR that does have 3 users, but since it's not explicitly a 1:1 room, it should behave like a group call for all practical purposes.